rasa/core/channels/slack.py

```diff
@@ -311,6 +329,18 @@ def _get_interactive_response(action: Dict) -> Optional[Text]:
 
         return response.text("")
 
+    def get_metadata(self, request: Request):
+        slack_event = request.json
+        event = slack_event.get("event")
+
+        metadata = {}
+        metadata["out_channel"] = event.get("channel")
```

Comment:
I just checked https://api.slack.com/types/event and there the `channel` param is part of `item`? Would also be awesome to add the link to https://api.slack.com/types/event as a comment in the code

Reply:
That's odd. This is the event object that I receive from the slack connector: 

``` javascript
{ 'client_msg_id': 'XXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX, 
  'type': 'message', 
  'text': 'hi', 
  'user': 'XXXXXXXXX', 
  'ts': '1579802617.000800', 
  'team': 'XXXXXXXXX', 
  'blocks': [{'type': 'rich_text', 'block_id': 'XXXXX', 'elements': [{'type': 'rich_text_section', 'elements': [{'type': 'text', 'text': 'hi'}]}]}], 
  'channel': 'XXXXXXXXX', 
  'event_ts': '1579802617.000800', 
  'channel_type': 'im'
}
```